src/EA.Weee.Database/scripts/Update/Release 5/Sprint03/20220420-1439-totals-stored-proc.sql

```diff
@@ -0,0 +1,59 @@
+﻿IF OBJECT_ID('[Evidence].[getAatfEvidenceSummaryTotals]', 'P') IS NOT NULL
+	DROP PROC [Evidence].[getAatfEvidenceSummaryTotals]
+GO
+
+CREATE PROCEDURE [Evidence].[getAatfEvidenceSummaryTotals]
+	@AatfId UNIQUEIDENTIFIER
+AS
+BEGIN
+SET NOCOUNT ON;
+
+IF OBJECT_ID('tempdb..#CategoryTotals') IS NOT NULL 
+BEGIN
+	DROP TABLE #CategoryTotals
+END
+
+CREATE TABLE #CategoryTotals(
+	CategoryId INT NOT NULL,
+	CategoryName NVARCHAR(500) NOT NULL,
```

Comment:
Although fairly minor, the datatype for [Lookup].[WeeeCategory].CategoryName is nvarchar(60), is the 500 just for safety?

Reply:
Good spot, updated.